core-tests/src/test/java/org/mule/runtime/core/processor/chain/DefaultMessageProcessorChainTestCase.java

```diff
@@ -87,34 +98,41 @@
 @RunWith(Parameterized.class)
 @SmallTest
 @SuppressWarnings("deprecation")
-public class DefaultMessageProcessorChainTestCase extends AbstractMuleContextTestCase {
+public class DefaultMessageProcessorChainTestCase extends AbstractReactiveProcessorTestCase {
```

Comment:
Me neither, hopefully you updates fixed it.  Otherwise, we need to change assertions to not depend on underlying reactor behaviour.